crates/edr_solidity/src/contracts_identifier/radix_tree.rs

```diff
@@ -0,0 +1,416 @@
+use std::collections::HashMap;
+
+#[derive(Debug)]
+pub struct RadixNode {
+    content: Vec<u8>,
+    is_present: bool,
+    bytes_matched_before: usize,
+    child_nodes: HashMap<u8, RadixNode>,
+}
+
+impl RadixNode {
+    fn new(content: Vec<u8>, is_present: bool, bytes_matched_before: usize) -> RadixNode {
+        RadixNode {
+            content,
+            is_present,
+            bytes_matched_before,
+            child_nodes: HashMap::new(),
+        }
+    }
+
+    pub fn get_content(&self) -> &Vec<u8> {
```

Comment:
```suggestion
    pub fn content(&self) -> &Vec<u8> {
```

In Rust getters don't prepend `get_` as it's redundant and complicates suggestions of the LSP (as all getters start with the same 3 letters)